proto/proto/thin_wallet.proto

```diff
@@ -14,7 +14,7 @@ service ThinWallet {
   rpc AssetLookup(crypto.AssetId) returns (chain.AssetInfo);
   rpc AssetList(AssetListRequest) returns (stream Asset);
   // TODO: return ValidatorStatus?
-  rpc ValidatorInfo(stake.IdentityKey) returns (stake.ValidatorDefinition);
+  rpc ValidatorStatus(stake.IdentityKey) returns (stake.ValidatorStatus);
```

Comment:
previously, the validator definition included the actual validator data, but now that the validator status just includes the identity key, we might want a second RPC...